bartop-api/src/api/user/user.graphql.js

```diff
@@ -1,17 +1,29 @@
 module.exports = `
+# Data model representing a user in BarTop
 type User {
-  id: String,
+
+  # Database-generated key to identify the User
+  id: ID,
+
+  # User ID provided by and used to interact with Auth0
   auth0Id: String
 }
 
+# Input type containing data needed to create a new User
 input UserInput {
+
+  # # User ID provided by and used to interact with Auth0
   auth0Id: String
```

Comment:
Should this be `ID` instead of `String` as well?

Reply:
Actually I guess it is `human`-readable so `String` makes sense 🤔

Reply:
Yeah I think String for sure for that